test/trento/application/integration/discovery/policies/sap_system_policy_test.exs

```diff
@@ -96,6 +121,30 @@ defmodule Trento.Integration.Discovery.SapSystemPolicyTest do
              |> SapSystemPolicy.handle([])
   end
 
+  test "should return the expected commands when a sap_system payload of type application and diagnostics is handled and the host is part of a cluster" do
```

Comment:
Do we need this test?
It doesn't test any new code as far as I see

Reply:
we are still in the same. I would remove this test, as it doesn't test anything new. But well, it is not harmful neither